src/Moryx.Model.InMemory/Moryx.Model.InMemory.csproj

```diff
@@ -5,6 +5,7 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <Description>InMemory extension for unit tests referencing Moryx.Model</Description>
     <CreatePackage>true</CreatePackage>
+    <PackageTags>ef6;Entity Framework;moryx;iiot;inmemory;iiot;iot</PackageTags>
```

Comment:
You use `Entity Framework` sometimes with spaces and sometimes without

e.g. "MORYX;Entity Framework;Data Model;Database;InMemory"

Reply:
Duplicate iiot tag